### PR TITLE
Fabo/added about page

### PR DESCRIPTION
**Pull request by faboweb**
First draft

Link:
image

Page:
image

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by fedekunze** [comment]
I wouldn’t put the actual address for security, also there’s a typo on the first line: should say develop instead of develope

File: app/src/renderer/components/common/PageAbout.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          The developers of Lunie accompanied the Cosmos SDK team to build
+          the amazing
+          tooling that exist today for building visual interfaces for Cosmos.
+          We want to continue assisting developement in the network so we all
+          can benefit. We want to keep Lunie <b>independent</b> from validators so
+          there is an entity you can trust.
+        </p>
+      </div>
+      <div class="card">
+        <h1>Contact</h1>
```
Comment (fedekunze) [inline app/src/renderer/components/common/PageAbout.vue]: I'd just leave the email and maybe a public Telegram/Reddit/Discord/Riot channel to reach out. Also I'd be nice to have the pictures of the team with media profiles.

Comment (faboweb) [reply app/src/renderer/components/common/PageAbout.vue]: Good ideas!

**Comment by faboweb** [comment]
Updated

**Comment by jbibla** [comment]
i'd like to add this to the footer instead of the sidebar link... but this might be hard without the fixes from #2353

**Comment by faboweb** [comment]
> i'd like to add this to the footer instead of the sidebar link... but this might be hard without the fixes from #2353

Let's do a footer when this is fixed?

**Pull request merged**